lib/python/qmk/cli/generate/config_h.py

```diff
@@ -73,6 +57,24 @@ def matrix_pins(matrix_pins, postfix=''):
     return '\n'.join(pins)
 
 
+def generate_matrix_size(kb_info_json, config_h_lines):
+    """Add the matrix size to the config.h.
+    """
+    if 'matrix_pins' in kb_info_json:
+        col_count = kb_info_json['matrix_size']['cols']
+        row_count = kb_info_json['matrix_size']['rows']
```

Comment:
Do you still remember why this code checks `'matrix_pins' in kb_info_json`, and not `'matrix_size' in kb_info_json`? With the `matrix_pins` check it is impossible to specify the matrix size in `info.json` if a custom matrix is used, and therefore `matrix_pins` is not defined.